.gitignore

```diff
@@ -12,3 +12,5 @@ assets/js/*.deps.json
 assets/js/*.map
 built
 /amphtml
+.env
+.idea/
```

Comment:
Might be something for your global `.gitignore` file :-)

Reply:
I think, it is helpful for devs that join the project.